### ISSUE TITLE
Message on popup when there are new addons

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#5073]
### Why this would be helpful/solve a problem

After a feature update (like 1.27.3 → 1.28.0), users generally have to manually enable any new addons. There is a [notification](https://user-images.githubusercontent.com/17484114/185762968-9c129851-5fef-421f-89c9-519acafcaabf.png) that appears once you visit Scratch to let you know, but what if some users don't see or read that?

### What this change does

My idea is to add a notification to the Addons page on the extension popup. The message at the top that displays "Explore all 140+ addons" could be changed to something like "New addons are available!" if the user hasn't seen the new addons yet.

### Possible alternatives

Could also make that notification more obvious, I suppose. Or display a purple badge icon (maybe with a + sign) on the extension icon to indicate that new addons were added, in addition to the aforementioned message.

There's no _super_ big need for all this to be clearer, I just thought it would be a nice user-experience enhancement.

### Demonstration(s) and additional context

![New addons are available. Go to the settings page to enable them. Open full screen settings.](https://user-images.githubusercontent.com/106490990/188231652-88ff3263-55c5-4f9c-8fe6-16d6c803df49.png)

**Comment by BroJac5246** [comment]
I agree, this would be great! I really dislike the "Check out all _x_+ addons" notification, but this would fix that problem by making it useful.

**Comment by Secret-chest** [comment]
I think it should be a tooltip attached to the cog icon saying something like "New addons available. Click to view and enable them." Clicking on the tooltip should press the cog icon, and the tooltip could animate

**Comment by DNin01** [comment]
> I think it should be a tooltip attached to the cog icon saying something like "New addons available. Click to view and enable them." Clicking on the tooltip should press the cog icon, and the tooltip could animate 

But how would the user come across that? No one would realize that there were new addons from a tooltip that they need to hover over something to see, so there should probably also be a badge on it in addition to this, or another solution.

**Comment by Secret-chest** [comment]
> > I think it should be a tooltip attached to the cog icon saying something like "New addons available. Click to view and enable them." Clicking on the tooltip should press the cog icon, and the tooltip could animate 
> 
> But how would the user come across that? No one would realize that there were new addons from a tooltip that they need to hover over something to see, so there should probably also be a badge on it in addition to this, or another solution.

I meant a permanent tooltip-style thingy that vanishes when you go to settings.

**Comment by DNin01** [comment]
> > > I think it should be a tooltip attached to the cog icon saying something like "New addons available. Click to view and enable them." Clicking on the tooltip should press the cog icon, and the tooltip could animate
> > 
> > 
> > But how would the user come across that? No one would realize that there were new addons from a tooltip that they need to hover over something to see, so there should probably also be a badge on it in addition to this, or another solution.
> 
> I meant a permanent tooltip-style thingy that vanishes when you go to settings.

Ah, so, like a notification right next to the cog.

**Comment by WorldLanguages** [comment]
After the v1.0.0 release, I didn't expect so many addons to be added! So I overlooked having something like this, and made an ugly notification popup when onion skinning launched, which was an epic new addon at the time. And not much has changed since then.
I also have always considered opening a new tab for every update to be invasive, even tho popular extensions sometimes do this, such as Tampermonkey and Wappalyzer.
I don't think this is high prio, but it's worth discussing.

**Comment by BroJac5246** [comment]
> After the v1.0.0 release, I didn't expect so many addons to be added! So I overlooked having something like this, and made an ugly notification popup when onion skinning launched, which was an epic new addon at the time. And not much has changed since then.
> I also have always considered opening a new tab for every update to be invasive, even tho popular extensions sometimes do this, such as Tampermonkey and Wappalyzer.
> I don't think this is high prio, but it's worth discussing. 

I like having the popup, since it provides easy access to the search bar. It could definitely be improved, though.
